### DISCUSSION TITLE
Dark theme ?

### DISCUSSION THREAD

**Discussion by h4ch3d** [nzbgetcom/nzbget#179] [General]
Hello, 

Is it planned to add a dark theme ?

Thanks for taking over the project.
H.

**Comment by luckedea** [comment]
@h4ch3d UI improvements are planned for v25 release.
It's not that trivial, as it is blocked by current version of bootstrap.

If this discussion gets more upvotes though, we'll focus on this sooner! :)

**Reply by h4ch3d** [reply]
Great !
Definitely a dark theme will give a more modern look ;)

Thanks.

**Reply by F1ghtClub** [reply]
+1 for Dark Theme. Thanks

**Reply by STRYKER69** [reply]
+1 for Dark Theme. Thanks too

**Comment by ydkmlt84** [comment]
I created this a long time ago and have not made any changes to it in a long time. I can't guarantee it even still works...I wanted to share anyways. https://github.com/ydkmlt84/DarkerNZBget

**Reply by luckedea** [reply]
Thank you, we might get some inspiration from this custom CSS. It's tempting to do a bigger update of the UI too.
It's fair to give it this task higher priority.

**Comment by h4ch3d** [comment]
I just installed testing version 23.1.
A big thank you for the integration of the dark theme.
The quick button to change the theme is a great idea.
Thanks for the improvements.
It's look perfect !
